sdk/appconfiguration/azure-spring-cloud-appconfiguration-config-web/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
+com.microsoft.azure.spring.cloud.config.web.AppConfigurationWebAutoConfiguration
```

Comment:
If we follow the track2 Spring guidelines, all Spring related packages should start with `com.azure.spring.`

Reply:
@saragluna I will add it to my list and update in the next preview version.